app/src/Pages/Portal/Dashboard/cards/SummaryCard.js

```diff
@@ -0,0 +1,216 @@
+    margin-bottom: 2px;
+  }
+
+  .money.big {
+    font-size: 3.2em;
+  }
+
+  .money.small {
+    font-size: 1.5em;
+  }
+
+  h4 {
+    font-size: 14px;
+    font-weight: normal;
+    margin-bottom: 0;
+    line-height: 1.1;
+  }
+
+  p {
+    margin-top: 0;
+  }
+
+  ul {
+    list-style: none;
+    padding-left: 0;
+  }
+
+  .loading-circle {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  @media ${mediaQueryRanges.mediumAndUp} {
+    .columns {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+    }
+    .column {
+      width: 45%;
+    }
+  }
+`;
+
+class SummaryCard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: true,
+      data: {},
+    };
+  }
+
+  getData() {
+    const { contributions = [], expenditures = [] } = this.props;
+
+    const contributionsOfStatus = status =>
+      contributions.find(item => item.status === status) || {
+        amount: 0,
+        matchAmount: 0,
+        total: 0,
+      };
+
+    const expendituresOfStatus = status =>
+      expenditures.find(item => item.status === status) || {
+        amount: 0,
+        total: 0,
+      };
+
+    const getTotals = contributionsOrExpenses => {
+      const result = {};
+      let total = 0;
+      contributionsOrExpenses.forEach(item => {
+        total += item.total;
+        // This is to capitalize the key 🙃
+        result[item.status[0].toUpperCase() + item.status.slice(1)] =
+          item.total;
+      });
+      result.total = total;
+      return result;
+    };
+
+    return {
+      totalContributions: getTotals(contributions),
+      totalExpenditures: getTotals(expenditures),
+      moneyRaised: contributionsOfStatus('Processed').amount,
+      moneyMatched: contributionsOfStatus('Processed').matchAmount,
+      moneyPending:
+        contributionsOfStatus('Draft').amount +
+        contributionsOfStatus('Submitted').amount,
+      moneySpentPending:
+        expendituresOfStatus('draft').amount +
+        expendituresOfStatus('submitted').amount,
+      moneySpentInCompliance: expendituresOfStatus('in_compliance').amount,
+      moneySpentOutOfCompliance: expendituresOfStatus('out_of_compliance')
+        .amount,
+    };
+  }
+
+  money(number) {
+    return `$${number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')}`;
+  }
+
+  renderStats() {
+    const { isLoading } = this.props;
+    const data = !isLoading ? this.getData() : {};
+
+    return (
+      <div className="columns">
+        <div className="column">
+          <h5>Contributions</h5>
+          <p className="top-section">
+            <Link to="/contributions">
+              Total: {data.totalContributions.total}
+            </Link>
+            {delete data.totalContributions.total}
+            {Object.keys(data.totalContributions).map(key => (
+              <Link to={`/contributions?status=${key.toLowerCase()}`}>
```

Comment:
I'm getting this error `Each child in a list should have a unique "key" prop.` despite there being a key.

Reply:
I pushed a fix that I think would fix this. Can you please pull and check?

Reply:
@billoosijok as long as it's okay that `draft` and `total` numbers are the same for now, this looks good. 👍

Reply:
Yeah that's what's coming from the backend for now